update-common/src/artifacts/update_plan.rs

```diff
@@ -276,8 +285,16 @@ impl<'a> UpdatePlanBuilder<'a> {
             data.extend_from_slice(&chunk);
         }
 
-        let (artifact_id, board) =
-            read_hubris_board_from_archive(artifact_id, data.clone())?;
+        // We purposely read out `name` instead of `board` here to
+        // allow for packaging multiple images for the same board
+        // in the TUF repo (specifically the manufacturing images)
+        // We will still select the image in wicketd via `board`
+        // as the only images we want to install via wicket are those
+        // with name == board.
```

Comment:
This seems correct but confusing (similarly for the related comment in `update_tracker`). Idly wondering: could wicket check here for `caboose.name == caboose.bord` and skip any images where that isn't true? I guess that'd just be `return Ok(())` without having actually done any work to add this image into any of the update plan maps? If so I think that would be equivalent but maybe slightly easier to reason about: _on ingest_, skip any images other than the ones that look like prod images, and then everything else continues to work in terms of board like it has.

Reply:
I think that's a good idea of the SP case but we'll still need to handle the RoTs as we do want to install some of the RoT images via wicket, specifically the bart signed images. 

I think what you're hitting on is we need to mark some artifacts  `noship`/`noupload` to indicate that wicketd should ignore them. That maybe a more complicated/nuanced long term problem.